packages/react-aria-components/test/DateField.test.js

```diff
@@ -194,6 +198,24 @@ describe('DateField', () => {
     expect(group).toHaveAttribute('data-validation-state', 'invalid');
   });
 
+  it('should support disabled render prop', () => {
```

Comment:
This is the test I was expecting, thank you. Now we just need it for TimeField.test.js as well.